packages/cli/generators/model/index.js

```diff
@@ -14,9 +14,10 @@ const path = require('path');
 
 const PROMPT_BASE_MODEL_CLASS = 'Please select the model base class';
 const ERROR_NO_MODELS_FOUND = 'Model was not found in';
-const BASE_MODELS = [
-  'Entity',
-  'Model',
+const BASE_MODELS = ['Entity', 'Model'];
```

Comment:
When generating the template, https://github.com/strongloop/loopback-next/blob/7be76a4c891d00875d4774d5ca2e3783f9ece53d/packages/cli/generators/model/templates/model.ts.ejs#L1 `isModelBaseBuiltin` checks if the model is in `BASE_MODELS` where the models are strings, but if I change it to `CLI_BASE_MODELS`, it won't find a match because `Entity` and `Model` are objects in that array.